scripts/review_comments_retriver.py

```diff
@@ -0,0 +1,205 @@
+        reraise=True,
+    )
+    def load_patch_set(self, diff_id):
+        try:
+            with open(f"patches/{diff_id}.patch", "r") as f:
+                patch = f.read()
+        except FileNotFoundError:
+            with open(f"patches/{diff_id}.patch", "w") as f:
+                patch = self.phabricator_api.load_raw_diff(diff_id)
+                f.write(patch)
+
+        return PatchSet.from_string(patch)
+
+    def _get_revisions_count(self):
+        return sum(1 for _ in phabricator.get_revisions())
+
+    def retrieve_diff_comments(self):
+        for revision in tqdm(
+            phabricator.get_revisions(), total=self._get_revisions_count()
+        ):
+            diff_comments: dict[int, list[InlineComment]] = defaultdict(list)
+
+            for transaction in revision["transactions"]:
+                if transaction["type"] != "inline":
+                    continue
+
+                # Ignore replies
+                if transaction["fields"]["replyToCommentPHID"] is not None:
+                    continue
+
+                # Ignore bot comments
+                if transaction["authorPHID"] == "PHID-USER-cje4weq32o3xyuegalpj":
+                    continue
+
+                if len(transaction["comments"]) != 1:
+                    print(transaction)
+                    # raise Exception("Unexpected, need to look into it")
+                    continue
+
+                comment_content = transaction["comments"][0]["content"]["raw"]
+
+                # Ignore very short and very log comments
+                if not 50 < len(comment_content) < 500:
+                    continue
+
+                # Ignore comments with URLs
+                if "https://" in comment_content or "http://" in comment_content:
+                    continue
+
+                #  Ignore nit comments
+                if self.NIT_PATTERN.search(comment_content):
+                    continue
+
+                # Ignore comments with code blocks
+                if "```" in comment_content:
+                    continue
+
+                comment_lower = comment_content.lower()
+                if any(
+                    phrase in comment_lower
+                    for phrase in [
+                        "wdyt?",
+                        "what do you think?",
+                        "you explain",
+                        "understand",
+                    ]
+                ):
+                    continue
+
+                diff_id = transaction["fields"]["diff"]["id"]
+                comment_id = transaction["comments"][0]["id"]
+                date_created = transaction["comments"][0]["dateCreated"]
+                filename = transaction["fields"]["path"]
+                start_line = transaction["fields"]["line"]
+                end_line = (
+                    transaction["fields"]["line"] + transaction["fields"]["length"] - 1
+                )
+                # Unfortantliy, we do not have this information for a limitation
```

Comment:
`Unfortunately`

Reply:
Fixed in d227893d7d02c4b9f1d0c1728741190e99b763d8. Spell check/auto-correct can sometimes make mistakes even worse :)